src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/NotificationPopUps.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.page
+
+import org.openqa.selenium.By
+import org.openqa.selenium.OutputType
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.remote.RemoteWebDriver
+
+internal class NotificationPopUps(private val driver: WebDriver) {
+    fun dismissHealthCheckNotifications() : NotificationPopUps {
+        // e.g. healthcheck notifications: "Don't remind me again"
+        return clickAll(By.cssSelector(".dismiss-notification"))
+    }
+
+    fun dismissAuiFlags() : NotificationPopUps {
+        // X mark on AUI flag
+        return clickAll(By.cssSelector(".aui-flag .icon-close"))
+    }
+    
+    fun dismissFindYourWorkFaster() : NotificationPopUps {
+        // Find your work faster "OK, got it"
+        return clickAll(By.cssSelector(".aui-button.helptip-close"))
+    }
+
+    fun disableNpsFeedback() : NotificationPopUps {
+        return clickAll(By.id("nps-acknowledgement-accept-button"))
+    }
+
+    fun dismissJiraHelpTips() : NotificationPopUps {
+        return clickAll(By.cssSelector(".jira-help-tip .cancel"))
+    }
+
+    fun dismissPostSetup() : NotificationPopUps {
+        return clickAll(By.className("postsetup-close-link"))
+    }
+
+    private fun clickAll(locator: By): NotificationPopUps {
+        driver.findElements(locator)
+            .filter { it.isEnabled && it.isDisplayed }
+            .forEach { 
+                try {
+                    it.click()
+                } catch (e: Exception) {
+                    println((driver as RemoteWebDriver).getScreenshotAs(OutputType.BASE64))
```

Comment:
Travis really hurts 😬
Could you mark these "artifacts via stdout" workarounds in comments or as a separate commit?

Reply:
I'll just remove it, let's hope they won't be needed 'anymore'.